packages/runtime-core/src/devtools.ts

```diff
@@ -28,7 +28,11 @@ interface DevtoolsHook {
   once: (event: string, handler: Function) => void
   off: (event: string, handler: Function) => void
   appRecords: AppRecord[]
-  _buffer: any[][]
```

Comment:
this version probably won't fix the leak for people using the older version of the devtools. Should we add a version that is compatible with both?

Reply:
For Vue 3 users have to use the new devtools. Updating should be done automatically (unless they use the electron app but they can update that too).